packager/media/base/aes_encryptor.cc

```diff
@@ -76,7 +68,14 @@ bool AesCtrEncryptor::CryptInternal(const uint8_t* plaintext,
 
   for (size_t i = 0; i < plaintext_size; ++i) {
     if (block_offset_ == 0) {
-      AES_encrypt(&counter_[0], &encrypted_counter_[0], aes_key());
+      size_t ignored_output_size;
+      uint8_t unused_iv[AES_BLOCK_SIZE];
+      CHECK_EQ(
+          mbedtls_cipher_crypt(&cipher_ctx_, unused_iv, AES_BLOCK_SIZE,
```

Comment:
nit: Does it work by setting iv to NULL and iv_size to 0? mbedtls implementation seems to ignore the IV for ECB mode anyway: https://siliconlabs.github.io/Gecko_SDK_Doc/mbedtls/html/cipher_8h.html#a1c249f6ee1a0d2c906927c7790c41dc5 and https://github.com/Mbed-TLS/mbedtls/issues/1091.

Reply:
I thought it was required, but I double-checked, and it's not.  This might have been left over from some older revision where I had the cipher context set up wrong.

Removed.  Thanks!